### ISSUE TITLE
Static page "spenden/fundraising"

### ISSUE THREAD

**Issue by anonjum** [hack4vdb/anthill-backend#92]
according to https://docs.google.com/document/d/19UIgaOo7MOyxIEx7Uh0nmeGAo5A5vs-E2OqT-ky7hgc/edit?usp=sharing

**Comment by stefan2904** [comment]
Also
- Header von vdb.at [Frontent, statisch]
- die Funktionalität von https://www.vanderbellen.at/spenden/ [schaut nach Frontent + JavaScript aus]
- \+ Progress (Kompliziert: wo bekommen wir den Stand her? soll eine Spende da direkt was triggern) [Frontent und Backend]
- viele Infotexte [Frontent, statisch]
- viele Sharebuttons [Frontent, JS]
- Footer von vdb.at [Frontent, statisch]

**Comment by anonjum** [comment]
Spendenstand und in der Seite spenden ist Feature request. 

stand müssten wir von vdb.at/Offenlegung parsen, dort wird veröffentlicht. 

Könnte man alle paar Tage Manuell eingeben. 

Rest ja.

**Comment by PeterTheOne** [comment]
@anonjum Auch hier: Soll https://www.vanderbellen.at/spenden/ ersetzen oder? Soll das nicht lieber erstmal in Typo3 bleiben?

**Comment by anonjum** [comment]
Nein, keinesfalls. Soll den mm Aspekt von fundraising abbilden und Leute animieren zu spenden / spenden zu sammeln.
